src/gsi/prewgt.f90

```diff
@@ -487,7 +487,8 @@ subroutine prewgt(mype)
   end do
 
 ! Special case of dssv for qoption=2 and cw
-  if (qoption==2) call compute_qvar3d
+  !if (qoption==2) call compute_qvar3d
```

Comment:
Is there a reason to save the commented line?
I am not familiar if `qoption /= 2` , supports the calling of this function.